packages/vue/src/components/sign-in.vue

```diff
@@ -87,7 +89,6 @@ const onForgotPasswordClicked = (): void => {
         </template>
         <federated-sign-in></federated-sign-in>
         <base-wrapper class="amplify-flex" style="flex-direction: column">
-          <slot name="header"></slot>
```

Comment:
I moved this _outside_ the form for consistency.

Plus, a header should be able to take up the full width of the container.

Reply:
👍